.github/workflows/linelint.yml

```diff
@@ -0,0 +1,13 @@
+on: pull_request
+name: linelint
+
+jobs:
+  linelint:
+    runs-on: ubuntu-latest
+    name: Check if all files end in newline
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

Comment:
`actions/checkout@v2` is a very legacy version, please use `v4` or add a reason for it.